doc/source/ray-core/fault_tolerance/ray_components.rst

```diff
@@ -0,0 +1,66 @@
+Failure of Ray components
+=========================
+
+Ray components failure could lead to tasks/objects/actors failure. 
+
+GCS failure
+-----------
+
+GCS failure usually means the Ray cluster failure and everything will be lost.
```

Comment:
"GCS failure usually means the Ray cluster failure and everything will be lost." is a bit awkward. Do you mean this in its severity?

The GCS failure is a single point of failure, meaning a Ray cluster failure where everything is lost.